.travis.yml

```diff
@@ -5,6 +5,7 @@ services:
 branches:
   only:
     - master
+    - rename-ocs-theme-variables
```

Comment:
remember to remove this after the build succeeds and we are ready to merge

Reply:
Just did for the merge.